src/components/Paper.jsx

```diff
@@ -43,20 +45,27 @@ export function Papers() {
                 de apresentar o teu paper aos participantes da JORDI!
               </p>
             </div>
-            <form className="ml-36">
-              <h3 className="text-lg font-semibold tracking-tight text-blue-900">
-                Submete o Paper aqui{' '}
-                <span className="hidden sm:inline-flex" aria-hidden="true">
-                  &darr;
-                </span>
-              </h3>
-              <Button type="submit" className="ml-20 mt-2 sm:ml-24" href="/">
-                <span className="sr-only sm:not-sr-only">Submissão</span>
-                <span className="sm:hidden">
-                  <ArrowRightIcon className="h-6 w-6" />
-                </span>
-              </Button>
-            </form>
+            <div className="ml-36 flex-col gap-10">
+              <form>
+                <h3 className="text-lg font-semibold tracking-tight text-blue-900">
+                  Submete o Paper aqui{' '}
+                  <span className="hidden sm:inline-flex" aria-hidden="true">
+                    &darr;
+                  </span>
+                </h3>
+                <Button type="submit" className="ml-20 mt-2 sm:ml-24" href="/">
+                  <span className="sr-only sm:not-sr-only">Submissão</span>
+                  <span className="sm:hidden">
+                    <ArrowRightIcon className="h-6 w-6" />
+                  </span>
+                </Button>
+              </form>
+              <div className="flex flex-row mt-16 items-center gap-3">
+                <Link className="font-mono text-xs tracking-tighter text-slate-600" href="/">Regulamentação</Link>
```

Comment:
I think these should be bigger and have some visual indication that they are links (like being underlined and/or a different color)